distribution/docs/src/main/resources/draft-documentation.adoc

```diff
@@ -0,0 +1,39 @@
+= DRAFT Distributed Data Framework Documentation
+include::${project.build.directory}/doc-contents/_contents/config.adoc[]
+
+== ${ddf-registry} Application Reference
```

Comment:
Apologies, if I missed it on another PR is there a reason you added the word "reference" to each application header?